spec/models/zone_spec.rb

```diff
@@ -192,6 +192,28 @@
     end
   end
 
+  context "maintenance zone" do
+    it "creates missing region during seed" do
+      expect(MiqRegion.my_region).to be_nil
+      described_class.seed
+      expect(MiqRegion.my_region).to be_present
+    end
+
+    it "is seeded with relation to region" do
+      described_class.seed
+      zone = Zone.where("name LIKE ?", "#{Zone::MAINTENANCE_ZONE_NAME_PREFIX}%").first
+
+      expect(described_class.maintenance_zone).to be_present
+      expect(described_class.maintenance_zone.id).to eq(zone.id)
```

Comment:
I don't think you need to compare `id`s, you can compare the objects themselves.

Reply:
comment above, right, but..